pkg/Microsoft.ML.AutoMLFeaturizers/Microsoft.ML.AutoMLFeaturizers.nupkgproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>ML.NET additional featurizers for AutoML</PackageDescription>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="../Microsoft.ML/Microsoft.ML.nupkgproj" />
+  </ItemGroup>
+
+  <ItemGroup> 
+    <Content Include="..\common\CommonPackage.props" Pack="true" PackagePath="build\netstandard2.0\$(MSBuildProjectName).props" />
```

Comment:
This line isn't necessary. This CommonPackage.props file probably should have a better name, but it is needed for NuGet packages that contain native assets inside of them. This one doesn't, so it is unnecessary.

Reply:
Sounds good. Removing in the next push.